content/new-routing.md

```diff
@@ -0,0 +1,446 @@
+2. How to define client and server routes for your app using Flow Router
+3. How to have your app display different content depending on the URL
+4. How to construct links to routes and go to routes programmatically
+
+## Client-side Routing
+
+In a web application, _routing_ is the process of using URLs to drive the user interface (UI). URLs are a prominent feature in every single web browser, and have several main functions from the user's point of view:
+
+1. **Bookmarking** - Users can bookmark URLs in their web browser to save content they want to come back to later
+2. **Sharing** - Users can share content with others by sending a link to a certain page
+3. **Navigation** - URLs are used to drive the web browser's back/forward functions
+
+In a traditional web application stack, where the server renders HTML one page at a time, the URL is the fundamental entry point for the user to access the application. Users navigate an application by clicking through URLs, which are sent to the server via HTTP, and the server responds appropriately via a server-side router.
+
+In contrast, Meteor operates on the principle of _data on the wire_, where the server doesn’t think in terms of URLs or HTML pages. The client application communicates with the server over DDP. Typically as an application loads, it boots up with a series of _subscriptions_ which fetch the data required to render the application. As the user interacts with the application, different subscriptions may load, but there’s no technical need for URLs to be involved in this process.
+
+However, most of the user-facing features of URLs listed above are still relevant for typical Meteor applications. Now that the server is not URL-driven, the URL just becomes a useful representation of the client-side state the user is currently looking at. However, unlike in a server-rendered application, it does not need to describe the entirety of the user’s current state; it simply needs to contain the parts that you want to be linkable. For example, the URL should contain any search filters applied on a page, but not necessarily the state of a dropdown menu or popup.
+
+## Using Flow Router
+
+To add routing to your app, install the [`kadira:flow-router`](https://atmospherejs.com/kadira/flow-router) package:
+
+```
+meteor add kadira:flow-router
+```
+
+Flow Router is a community routing package for Meteor. At the time of writing this guide, it is at version 2.x.
+
+- [Flow Router on GitHub](https://github.com/kadirahq/flow-router)
+- [Kadira Meteor routing guide](https://kadira.io/academy/meteor-routing-guide)
+
+### Other options for routing
+
+Flow Router is one of several popular routing packages for Meteor. Another is iron:router. You can search for router on Atmosphere to find more. Hopefully, the concepts in this routing guide will be relevant no matter which router you use, as long as it provides basic functions for URL management.
```

Comment:
I think we should be a little more forceful here to say that really you should really be using Flow Router unless you have a good reason not to.